src/main/java/core/basesyntax/MainApp.java

```diff
@@ -2,4 +2,19 @@
 
 public class MainApp {
 
+    public static void main(String[] args) {
+
```

Comment:
Remove the redundant empty line after the method signature as per the checklist guidelines.